app/javascript/packages/time-element/index.spec.ts

```diff
@@ -53,12 +53,15 @@ describe('TimeElement', () => {
     usePropertyValue(Intl.DateTimeFormat.prototype, 'formatToParts', undefined);
 
     it('sets text using Intl.DateTimeFormat#format as fallback', () => {
+      const date = new Date(2020, 3, 21, 14, 3, 24);
       const element = createElement({
         format: '%{month} %{day}, %{year} at %{hour}:%{minute} %{day_period}',
-        timestamp: new Date(2020, 3, 21, 14, 3, 24).toISOString(),
+        timestamp: date.toISOString(),
       });
 
-      expect(element.textContent).to.equal('April 21, 2020, 2:03 PM');
+      const expected = element.formatter.format(date);
+
+      expect(element.textContent).to.equal(expected);
```

Comment:
I liked the readability of the old one. Could we do something like this?
```suggestion
      expect([
        "April 21, 2020, 2:03 PM", // node 14
        "April 21, 2020 at 2:03 PM", // node 16
      ]).to.include(element.textContent);
```

Reply:
We could. I agree on the point about general readability, but I also think we don't care about the specific text, we just care that it matches the output of `Intl.DateTimeFormat#format`, per the test description.

Reply:
ah ok, makes sense